docs/release_notes/v1.7.0.md

```diff
@@ -0,0 +1,386 @@
+
+##### Upgrade using Helm
+
+To upgrade Dapr using Helm, run:
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+
+helm upgrade dapr dapr/dapr --version 1.7.0 --namespace=dapr-system --wait
+```
+
+Wait until the operation is finished and check your status with `dapr status -k`.
+
+All done!
+
+*Note: Make sure your deployments are restarted to pick the latest version of the Dapr sidecar*
+
+#### Starting a fresh install on a cluster
+
+Please see [how to deploy Dapr on a Kubernetes cluster](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/) for a complete guide to installing Dapr on Kubernetes
+
+You can use Helm 3 to install Dapr:
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+
+kubectl create namespace dapr-system
+
+helm install dapr dapr/dapr --version 1.7.0 --namespace dapr-system --wait
+```
+
+Alternatively, you can use the latest version of CLI:
+
+```
+dapr init --runtime-version=1.7.0 -k
+```
+
+##### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ dapr status -k
+  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
+  dapr-dashboard         dapr-system  True     Running  1         0.10.0    15s  2022-03-29 13:07.39
+  dapr-sidecar-injector  dapr-system  True     Running  1         1.7.0   15s  2022-03-29 13:07.39
+  dapr-sentry            dapr-system  True     Running  1         1.7.0   15s  2022-03-29 13:07.39
+  dapr-operator          dapr-system  True     Running  1         1.7.0   15s  2022-03-29 13:07.39
+  dapr-placement         dapr-system  True     Running  1         1.7.0   15s  2022-03-29 13:07.39
+```
+
+After Dapr 1.7.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Components
+- In this release the `value` field has been removed from the key name in the state management Query API as part of the ongoing feedback on this preview feature.
+    - **FIXED** statestore preview Query API: Remove 'value' prefix from the key names in query JSON [1490](https://github.com/dapr/components-contrib/issues/1490)
+- This release include the breaking change to the MongoDB store [as described in v1.6.1 release notes](https://github.com/dapr/dapr/releases/tag/v1.6.1) to address a data serialization problem.
+  - **FIXED** MongoDB state store: Removes MongoDB bson artifacts from JSON value output [1582](https://github.com/dapr/components-contrib/issues/1582)
+
+### Java SDK
+- **REMOVED** Deprecated builder classes [601](https://github.com/dapr/java-sdk/issues/601)
+
+## Deprecation Notices
+- Content-type will not longer default to `application/json` if not explictly provided as a content-type header value for HTTP invocations starting in Dapr 1.9. Users are recommended to turn on the 
```

Comment:
this line sounds incomplete ...

Reply:
argh it got cut off somehow

Reply:
It looks weird because it's referring to the feature flag PR. maybe should link to docs instead. I didn't write that

Reply:
This is the point that I was asking in the chat. Where is the docs link to the new feature flag?

Reply:
It's already updated